Update tendrl-ansible for Tendrl Milestone #3

mbukatov opened the pull request:
Addressing https://github.com/Tendrl/tendrl-ansible/issues/23

Work in progress, but all new configuration and setup steps should be already included.

There is one known issue which makes this installation fail though: https://github.com/Tendrl/monitoring-integration/issues/33

mbukatov committed:
tendrl-copr: selecting buils from master (optional)

Release COPR is still the default, but for development and testing
purposes, one can chose to install latest snapshot builds from master
branch.

mbukatov committed:
tendrl-monitoring-integration is now service

Based on https://github.com/Tendrl/specifications/issues/233

cybernth commented:
@mbukatov Please have a look -  https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference/_compare/39a453a0f15eacb84c90bc97df50f8aef7f091d9...2daa903c86f978413d815052f3cfbd6dcb754f74
Please take care of the updates on the storage node install/configure

mbukatov commented:
@nthomas-redhat ack

mbukatov committed:
site.yml.sample: drop gluster provisioning

Based on Nishanth's update of Tendrl Installation Guide[1]

[1] https://github.com/Tendrl/documentation/wiki/Tendrl-Package-Installation-Reference/_compare/39a453a0f15eacb84c90bc97df50f8aef7f091d9...2daa903c86f978413d815052f3cfbd6dcb754f74
